components/admin_console/system_users/list/selectors.jsx

```diff
@@ -0,0 +1,38 @@
+import {getUser, getProfiles, getProfilesInTeam, getProfilesWithoutTeam, searchProfiles, searchProfilesInTeam} from 'mattermost-redux/selectors/entities/users';
```

Comment:
Need to add missing header (copyright and license info) :)

Reply:
Gah! And that after just fixing the above...